manager-bundle/skeleton/config/config.yaml

```diff
@@ -31,10 +31,26 @@ framework:
                 adapter: cache.app
             doctrine.system_cache_pool:
                 adapter: cache.system
+    messenger:
```

Comment:
The way to read this configuration:

- 3 interfaces map to 3 transports
- These 3 transports are configured to use our own `contao_cron_fallback` transport. All of them are configured to use the doctrine transports and fallback to `sync`.